### PR TITLE
Fix polyfills (included in CMS app)

### LINKED ISSUE #695: CMS broken in IE11
**- Do you want to request a *feature* or report a *bug*?**
bug

**- What is the current behavior?**
The CMS seems to be broken in IE 11. The error deals with `Symbol` not being a valid type.

**- If the current behavior is a bug, please provide the steps to reproduce.**

**- What is the expected behavior?**
The CMS should work in IE 11.

**- Please mention your CMS, node.js, and operating system version.**

master

### PR DESCRIPTION
**Pull request by talves**
**- Summary**

Fix the polyfill issue with IE11 and any supported browsers

Closes #695

**- Description for the changelog**

Bundles the polyfills into the app.
Uses the `env` babel solution to identify the shims needed in the bundle.
Is the alternative solution to separation in #698

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,11 +2,11 @@ import React from 'react';
 import createReactClass from 'create-react-class';
 import { render } from 'react-dom';
 import { AppContainer } from 'react-hot-loader';
-import 'file-loader?name=index.html!../example/index.html';
```
Comment (tech4him1) [inline src/index.js]: I think this should be a separate PR, since it doesn't deal directly with this.

File: .babelrc [hunk 2]
```diff
@@ -1,5 +1,12 @@
 {
-  "presets": [["es2015", { "modules": false }], "stage-1", "react"],
+  "presets": [
+    ["env", {
+      "modules": false,
+      "useBuiltIns": false,
```
Comment (tech4him1) [inline .babelrc]: Does this need to be explicitly specified, since it is the default anyway?

File: src/lib/polyfills/index.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+import 'babel-polyfill';
```
Comment (tech4him1) [inline src/lib/polyfills/index.js]: I wonder if we really need to import the whole polyfill, or just the parts that we need? For right now we should probably just merge as-is, though.

File: .babelrc (hunk 2 above)
Comment (talves) [reply .babelrc]: True. It showed what we were attempting and I made the fix from your first attempt 😄 from "usage"

File: src/lib/polyfills/index.js (hunk 3 above)
Comment (talves) [reply src/lib/polyfills/index.js]: Babel only includes what is needed from the code, why would we try to determine that ourselves. right?

Comment (tech4him1) [reply src/lib/polyfills/index.js]: I'm still thinking about bundle size, not runtime checking.

File: .babelrc (hunk 2 above)
Comment (tech4him1) [reply .babelrc]: Good point, just leave it in there, then.

File: src/lib/polyfills/index.js (hunk 3 above)
Comment (talves) [reply src/lib/polyfills/index.js]: I don't think it will change the determination for bundle size in the end.

Comment (tech4him1) [reply src/lib/polyfills/index.js]: > Babel only includes what is needed from the code, why would we try to determine that ourselves. right?

If you set `useBuiltIns: true` above, then yes, that is true.

File: .babelrc (hunk 2 above)
Comment (talves) [reply .babelrc]: Could use include and exclude:
```
["env", {
      "targets": {
        "browsers": ["last 2 versions", "safari >= 7"]
      },
      "include": ["transform-es2015-arrow-functions", "es6.map"],
      "exclude": ["transform-regenerator", "es6.set"]
    }]
```

Comment (tech4him1) [reply .babelrc]: What would you think that would change?

Comment (erquhart) [reply .babelrc]: I like using a centralized browserslist in package.json.

File: src/index.js (hunk 1 above)
Comment (talves) [reply src/index.js]: ok, see #701
